docs/release-notes.md

```diff
@@ -1,3 +1,23 @@
+## Release v0.5.0
+### Date: April 27, 2023
+This release introduces the Metadata Schema Service and the Policy Service API. Release Automations have been expanded to include Pass-Acceptance-Testing and Pass-Docker.
+
+* More release automations added to include Pass-Acceptance-Testing and Pass-Docker
```

Comment:
These three bullets really just repeat the sentence above, so aren't that helpful.